### PR TITLE
Android commit pt2

### PR DESCRIPTION
**Pull request by jeremybeal11**
convert to pocketEth

### CODE REVIEW

File: ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/CreateWalletActivity.java [hunk 1]
```diff
@@ -53,14 +60,18 @@ public void onClick(View cs) {
...
     }
-
     protected void createWallet() {
-        wallet = this.pocketAion.getMastery().createWallet();
+        wallet = this.pocketEth.getRinkeby().createWallet();
+
+        publicView.setText(wallet.getAddress());
+        privateView.setText(wallet.getPrivateKey());
+
+        Log.d("testing", wallet.getAddress());
```
Comment (pabelnl) [inline ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/CreateWalletActivity.java]: Remove this log

File: ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/ImportWalletActivity.java [hunk 2]
```diff
@@ -100,10 +101,12 @@ public void onClick(DialogInterface dialog, int id) {
                             wallet.save(passphrase, ImportWalletActivity.this, new Function1<WalletPersistenceError, Unit>() {
                                 @Override
                                 public Unit invoke(WalletPersistenceError walletPersistenceError) {
-                                    ImportWalletActivity.this.loadMessagesActivity();
```
Comment (pabelnl) [inline ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/ImportWalletActivity.java]: why it was removed from the completion handler?

File: ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/MessageScreenActivity.java [hunk 3]
```diff
@@ -24,9 +24,9 @@
 
 import kotlin.Unit;
 import kotlin.jvm.functions.Function2;
-import network.pocket.aion.AionContract;
-import network.pocket.aion.PocketAion;
-import network.pocket.aion.exceptions.AionContractException;
+import network.pocket.eth.EthContract;
+import network.pocket.eth.*;
+import network.pocket.eth.exceptions.EthContractException;
```
Comment (pabelnl) [inline ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/MessageScreenActivity.java:29]: exceptions are already included with the previous line eth.*

File: ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/MessageScreenActivity.java [hunk 4]
```diff
@@ -134,7 +134,7 @@ public void run() {
     public void retrieveMessages() {
         // Retrieve messages count
         try {
-            this.aionContract.executeConstantFunction("getTotalMessageCount", null, null, new BigInteger("50000"), new BigInteger("20000000000"), null, new Function2<PocketError, Object[], Unit>() {
+            this.ethContract.executeConstantFunction("getTotalMessageCount", null, null, new BigInteger("50000"), new BigInteger("20000000000"), null, new Function2<PocketError, Object[], Unit>() {
```
Comment (pabelnl) [inline ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/MessageScreenActivity.java]: Does the values for gas and gasPrice works? is this tested?

File: ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/MessageScreenActivity.java [hunk 5]
```diff
@@ -164,7 +164,7 @@ protected void getMessageWith(Integer index) {
         ArrayList<Object> functionParams = new ArrayList<>();
         functionParams.add(new BigInteger(index.toString()));
         try {
-            MessageScreenActivity.this.aionContract.executeConstantFunction("getMessageByIndex", functionParams, null, new BigInteger("50000"), new BigInteger("20000000000"), null, new Function2<PocketError, Object[], Unit>() {
+            MessageScreenActivity.this.ethContract.executeConstantFunction("getMessageByIndex", functionParams, null, new BigInteger("50000"), new BigInteger("20000000000"), null, new Function2<PocketError, Object[], Unit>() {
```
Comment (pabelnl) [inline ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/MessageScreenActivity.java]: gas and gasPrice values are correct? values varies for every blockchain

File: ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/smartcontract/SmartContract.java [hunk 6]
```diff
@@ -14,23 +14,25 @@
 
 import kotlin.Unit;
 import kotlin.jvm.functions.Function2;
-import network.pocket.aion.*;
-import network.pocket.aion.exceptions.AionContractException;
+import network.pocket.eth.*;
+import network.pocket.eth.exceptions.EthContractException;
```
Comment (pabelnl) [inline ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/smartcontract/SmartContract.java:18]: remove this line

**Review by pabelnl — CHANGES_REQUESTED** [review]

File: ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/MessageScreenActivity.java (hunk 3 above)
Comment (jeremybeal11) [reply ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/MessageScreenActivity.java:29]: i tried, but it won't include the exception, so i had to manually import it.

File: ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/MessageScreenActivity.java (hunk 5 above)
Comment (jeremybeal11) [reply ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/MessageScreenActivity.java]: just changed it to the correct gas amount. check new commit

**Review by pabelnl — APPROVED** [review]

**Pull request merged**